index.html

```diff
@@ -1670,7 +1670,7 @@ <h5 class="modal-title-craft"></h5>
                                                         <input type="text" step="1" min="0" max="999.00" />
                                                     </td>
                                                     <td name="dynNotchMaxHz">
-                                                        <label>>Max (Hz)</label>
+                                                        <label>Max (Hz)</label>
```

Comment:
Not needed, already fixed in  #555 already fixed the '>>' here.
Maybe rebase when possible?

Reply:
I still need to include some semantic versioning or do we not use this in the blackbox explorer?